rome_app/templates/rome_templates/page_detail.html

```diff
@@ -85,6 +84,21 @@
     div.intro, div.pagination_rome {
         display: none;
     }
+    .page_icon {
+        background: url(../../static/rome/images/prints.png);
```

Comment:
This isn't working on the test server. You can change this to something like the following:
background: url({% static 'rome/images/prints.png' %});

You can do that other places your reference a static file as well. Note that you need the following line at the top of the page if it's not there already:
{% load static from staticfiles %}